helm/defectdojo/templates/_helpers.tpl

```diff
@@ -46,14 +46,16 @@ Create the name of the service account to use
   Determine the hostname to use for PostgreSQL/Redis.
 */}}
 {{- define "postgresql.hostname" -}}
-{{- if .Values.postgresql.enabled -}}
-{{- if eq .Values.postgresql.architecture "replication" -}}
-{{- printf "%s-%s-%s" .Release.Name "postgresql" .Values.postgresql.primary.name | trunc 63 | trimSuffix "-" -}}
+{{- if .Values.cloudsql.enabled -}}
+{{-  printf "127.0.0.1" -}}
+{{- else if .Values.postgresql.enabled -}}
+{{-  if eq .Values.postgresql.architecture "replication" -}}
+{{-   printf "%s-%s-%s" .Release.Name "postgresql" .Values.postgresql.primary.name | trunc 63 | trimSuffix "-" -}}
+{{-  else -}}
+{{-   printf "%s-%s" .Release.Name "postgresql" | trunc 63 | trimSuffix "-" -}}
+{{-  end -}}
 {{- else -}}
-{{- printf "%s-%s" .Release.Name "postgresql" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-{{- else -}}
-{{- printf "%s" .Values.postgresql.postgresServer -}}
+{{-  printf "%s" .Values.postgresql.postgresServer -}}
```

Comment:
Isn't it easier and more future-proof (in case some other provider is implemented) to replace this line with? 
```suggestion
{{-  printf "%s" .Values.postgresql.postgresServer | default "127.0.0.1" -}}
```